### PR TITLE
architecture/additional_concepts/authorization: improvements to the list of allowed volume types

### PR DESCRIPTION
**Pull request by php-coder**
- add missing volume types
- mention "none" volume type
- add links to the Kubernetes documentation for the volume types
- fix field name
- add note that the list can be outdated
- add "projected" to the list of recommended minimum of allowed volumes

@pweil- @liggitt PTAL  and let me know if we don't want to document "none" value.

CC @simo5

### CODE REVIEW

**Comment by liggitt** [comment]
LGTM

**Comment by php-coder** [comment]
PTAL @openshift/team-documentation

**Comment by ahardin-rh** [comment]
LGTM. Looks like this is targeting 3.6, correct?

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -551,14 +551,14 @@ The recommended minimum set of allowed volumes for new SCCs are *configMap*,
 
 [NOTE]
 ====
-`***` is a special value to allow the use of all volume types.
+`***` is a special value to allow the use of all volume types. `*none*` is a special value to disallow the use of all volumes types (exist only for backward compatibility).
```
Comment (mburke5678) [inline architecture/additional_concepts/authorization.adoc]: Should *none* be in the list above, under *** ?

**Comment by mburke5678** [comment]
@php-coder One question. Otherwise LGTM

**Comment by php-coder** [comment]
> LGTM. Looks like this is targeting 3.6, correct?

Correct. (The field name can by applied to other branches...)

File: architecture/additional_concepts/authorization.adoc (hunk 1 above)
Comment (php-coder) [inline architecture/additional_concepts/authorization.adoc]: Yes, I can put it there.

Comment (php-coder) [inline architecture/additional_concepts/authorization.adoc]: Done. I also add more unrelated changes that I found.

**Comment by php-coder** [comment]
PTAL again, I've added many minor improvements.

File: architecture/additional_concepts/authorization.adoc [hunk 2]
```diff
@@ -521,44 +521,50 @@ The usage of specific volume types can be controlled by setting the `volumes`
...
 
 The recommended minimum set of allowed volumes for new SCCs are *configMap*,
-*downwardAPI*, *emptyDir*, *persistentVolumeClaim*, and *secret*.
+*downwardAPI*, *emptyDir*, *persistentVolumeClaim*, *secret*, and *projected*.
 
 [NOTE]
 ====
-`***` is a special value to allow the use of all volume types.
+The list of allowable volume types could be not full because the new types are
+adding in each release of {product-title}.
```
Comment (mburke5678) [inline architecture/additional_concepts/authorization.adoc]: The list of allowable volume types is not exhaustive because new types are added with each release of {product-title}.

Comment (mburke5678) [inline architecture/additional_concepts/authorization.adoc]: Can we keep the list updated with each release, so that the list is complete?

**Comment by mburke5678** [comment]
@php-coder One comment and question

File: architecture/additional_concepts/authorization.adoc (hunk 2 above)
Comment (php-coder) [inline architecture/additional_concepts/authorization.adoc]: > Can we keep the list updated with each release, so that the list is complete?

Yes, perhaps, but who will responsible to do that each release?

**Comment by php-coder** [comment]
@mburke5678 Comment addressed. PTAL.

**Comment by mburke5678** [comment]
LGTM!

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by ahardin-rh** [comment]
No revision history needed
